README.md

```diff
@@ -3,18 +3,47 @@ A list of resources for data science work on climate change
 
 ## Data Sets
 
+### Global
+* [ICE Embodied Carbon Database](https://circularecology.com/embodied-carbon-footprint-database.html)
+* [Carbon in food downloadable csv](https://ourworldindata.org/food-choice-vs-eating-local)
+* [EPW Weather files (for Ladybug on Grasshopper, but adaptable)](https://www.ladybug.tools/epwmap/)
```

Comment:
Instead of a single link to a hosted tool, can this be a link to their GH account, and put under the companies heading please :) https://github.com/ladybug-tools

Reply:
This is a link to the front end of the data library of several thousand weather data packs around the world.